README.md

```diff
@@ -44,11 +44,24 @@ The first line of the newly created file will contain the link to the selected e
+Once you've solved the exercise, it's easy to submit your solution. There are three ways to submit a solution:
 
-![Submit](assets/submit2.png)
+* Click the checkmark ( <img src="assets/content-page-read.svg" alt="checkmark" width=15px;/> ) in the left panel with the Dodona activities.![Checkmark](assets/check-mark.png)
+* You can also use the `Alt+D` shortcut (`Option-D` for Mac users) for this, or assign a [different shortcut](https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-shortcuts-editor).
+* Another option is to execute the `Dodona: Submit solution`-command by opening the command palette using `Ctrl+Shift+P` and typing (parts of) `Dodona: Submit solution` and pressing `Enter`.![Submit](assets/submit2.png)
 
-After a few seconds, you should see a popup in the bottom right corner containing the result of your submission. If you click the `View results` button, the result should open in a new browser window.
+This command will submit your code to Dodona and trigger an automatic test.
+
+After a few seconds, you should see a popup in the bottom right corner containing the result of your submission. If you click the `View results` button, the result with the difference between your output and the expected output should open in a new browser window.
+
+## Other functionality
+|                                 Symbol                                 | Meaning                                                                                                                                                                               |
+| :--------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|       <img src="assets/refresh.svg" alt="refresh" width=15px;/>        | Reload the list of activities.                                                                                                                                                        |
+|  <img src="assets/notification.svg" alt="notification" width=15px;/>   | Notifies users in case of unread feedback. The icon links to https://dodona.ugent.be/notifications.                                                                                   |
+| <img src="assets/content-page-read.svg" alt="checkmark" width=15px;/>  | Submit solution to Dodona. Alternative for `Alt+D` and `Ctrl+Shift+P`, `Dodona: Submit Solution`.                                                                                     |
+|          <img src="assets/edit.svg" alt="edit" width=15px;/>           | Opens the description of an exercise series or course in a new browser window. (Hover with your cursor above an exercise series or course in the left column with Dodona activities.) |
```

Comment:
Use commas when summing up, hover `over`
```suggestion
|          <img src="assets/edit.svg" alt="edit" width=15px;/>           | Opens the description of an exercise, series or course in a new browser window. (Hover with your cursor over an exercise, series or course in the left column with Dodona activities.) |
```

Reply:
I meant "oefeningenreeks" in Dutch. Does this translate to `exercise series`?

Reply:
Oh, okay. I'll edit my suggestion.

In that case: both Dodona itself and our plugin refer to these as `series` everywhere, so for the sake of consistency I feel like it's better to stick with that. If you look at the URLs you'll also notice this.

`dodona.ugent.be/series/XXX`, `dodona.ugent.be/courses/XXX/series`, etc.

Reply:
I searched for it here https://dodona-edu.github.io/en/guides/teachers/exercise-series-management/ but there was no English translation available of that page. Thank you for your clarification.